models/pokemon_model.go

```diff
@@ -0,0 +1,6 @@
+package models
```

Comment:
Same thing about package and file naming here, singular and no need for the suffix